src/main/resources/application.yaml

```diff
@@ -16,5 +16,11 @@ info:
     description: Ingest scanned documents.
 
 storage:
-  account_name: ${STORAGE_ACCOUNT_NAME}
-  key: ${STORAGE_KEY}
+  account_name: ${STORAGE_ACCOUNT_NAME:bulkscanning}
+  key: ${STORAGE_KEY:dGVzdA==} #Base 64 encoded
+
+accesstoken:
+  serviceConfig:
+    - serviceName: sscs
+      permissions: WRITE,LIST
```

Comment:
should this actually be configurable?

Reply:
I had a chat with Salman about this and he thinks it's better to make it configurable as we can remove/add permissions through config file rather than changing source code.Also a PR need to be created and approved before anyone updates the permissions so that way it won't go wrong.

Reply:
I know we had a chat about it in the morning but what I got to know is LIST permission could be removed later that is one of the requirement.Alternatively what we are gaining hardcoding it ? I think config makes it easier to change rather than code.

Reply:
Because this service should never dispense a SAS with READ permissions; that's a security requirement.

If we hard code the permissions and add comments explaining why, then we make this clear.

If we instead make these configurable, the implication is that they can be changed arbitrarily. 

When I see something is configurable, I assume there must be a legitimate scenario that would justify changing it, or why would it be configurable?

Reply:
Also, we cannot remove LIST permissions since Exela would not be able to use Azure's utilities to upload files.

Reply:
They can't be changed arbitrarily.You will need to update the config and deploy isn't it ? How come it can be changed arbitrarily.

Reply:
You only need write permissions to upload files.

Reply:
Bulk copy tools need to be able to LIST files to see which files they need to upload, and which have already been uploaded.

The fact that the config has to be changed by PR is irrelevant; these permissions are constants and should be coded as such - it is misleading to code anything else; you wouldn't put PI in a configuration file, so why would you put these permissions?